docs/core/track_fitting.md

```diff
@@ -1 +1,33 @@
 # Track Fitting
+
+## Gaussian Sum Filter
+
+The GSF is an extension of the Kalman-Filter that allows to handle non-gaussian errors by modelling the track state as a gaussian mixture:
+
+$$
+p(\vec{x}) = \sum_i w_i \varphi(\vec{x}; \mu_i, \Sigma_i), \quad \sum_i w_i = 1
+$$
+
+In particular this is used in Acts for electron refitting. The energy-loss of Brehmsstrahlung for electrons in matter are highly non-gaussian, and thus are not modeled accurately by the default material interactions in the Kalman Filter. Instead, the Brehmsstrahlung is modeled as a Bethe-Heitler distribution, which is approximated as a gaussian mixture. For further reading see [here](https://cds.cern.ch/record/1448253).
+
+### Implementation
+
+To implement the GSF, a special stepper is needed, that can handle a multi-component state internally: The {class}`Acts::MultiEigenStepperLoop`. On a surface with material, for each component the material loss is modeled with a approximated Bethe-Heitler distribution with a fixed number of components. Since the number of components would grow exponentially with each material interaction, close components are merged afterwards to limit the computational cost.
```

Comment:
not sure if my suggestion makes sense

```suggestion
To implement the GSF, a special stepper is needed, that can handle a multi-component state internally: The {class}`Acts::MultiEigenStepperLoop`. On a surface with material, the Bethe-Heitler loss distribution is approximated  with a fixed number of gaussian components. Since the number of components would grow exponentially with each material interaction, components in close proximity are merged to limit the computational cost.
```